src/Microsoft.Diagnostics.NETCore.Client/DiagnosticsIpc/IpcClient.cs

```diff
@@ -122,5 +120,17 @@ private static IpcMessage Read(Stream stream)
         {
             return IpcMessage.Parse(stream);
         }
+
+        private static EndPoint CreateUnixDomainSocketEndPoint(string path)
+        {
+#if NETCOREAPP
+            return new UnixDomainSocketEndPoint(path);
+#elif NETSTANDARD2_0
+            // UnixDomainSocketEndPoint is not part of .NET Standard 2.0
+            var type = typeof(Socket).Assembly.GetType("System.Net.Sockets.UnixDomainSocketEndPoint");
+            var ctor = type.GetConstructor(new[] { typeof(string) });
+            return (EndPoint)ctor.Invoke(new object[] { path });
+#endif
```

Comment:
nit: add an `#else` statement and with a descriptive [#error](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-error) directive.